setup.sh

```diff
@@ -15,5 +15,6 @@ else
     echo "Could not verify system is RedHat or Debian."
     exit 1
 fi 
-
+gem sources --add http://gems.ruby-china.org/ --remove https://rubygems.org/
```

Comment:
Why push these changes?
That's probably nice for Chinese contributors, but not for the whole world.

Reply:
looks weird to me too.

Reply:
I guess it's for speeding up local build in china. Please drop these changes from the pull request.